### PR TITLE
Fix ppc64le binary

### PR DESCRIPTION
**Pull request by leofang**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by leofang** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,8 +9,9 @@ source:
   sha256: ce128ea090b05e36d00ffe921e45982ca10e8207e40cfc2e0067d0f62d9b36f9  # [ppc64le and cuda_compiler_version == "10.2"]
 
 build:
-  number: 2
+  number: 3
   skip: True   # [(not ppc64le) or (cuda_compiler_version != "10.2")]
+  rpaths_patcher: LIEF  # [ppc64le]
```
Comment (IvanYashchuk) [inline recipe/meta.yaml:14]: Interesting, are there any benefits of using LIEF instead of PatchELF for other builds as well?

**Comment by IvanYashchuk** [comment]
There is an open issue about LIEF's interaction with ppc64le https://github.com/lief-project/LIEF/issues/304.

**Comment by leofang** [comment]
OK so switching to LIEF works! Let's merge this as a workaround if we don't hear back about the other alternative https://github.com/conda-forge/patchelf-feedstock/pull/20 by tomorrow.

**Comment by leofang** [comment]
patchelf 0.12+ fixed this issue too, so I've switched back to patchelf. Let's get this in!

**Comment by IvanYashchuk** [comment]
Thank you, @leofang, for fixing this!

**Pull request merged**